src/components/Friends/Friend/index.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect, memo } from 'react'
+import './styles.scss'
+import propTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+
```

Comment:
Consider using some kind of automatic formatting plugin together with prettier rules, makes life so much easier :) Some empty lines and missing spaces in this file.